config/app.php

```diff
@@ -15,6 +15,17 @@
 
     'name' => env('APP_NAME', 'Monica'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Application Display Name
+    |--------------------------------------------------------------------------
+    |
+    | This value is the name of your application displayed in email notifications.
```

Comment:
`This is the name of the application that will be displayed in the notification emails`.